transformers/events/DOCUMENTATION.md

```diff
@@ -2,11 +2,13 @@
 
 ## Architecture
 
-Transformers fetch logs from Ethereum, convert/decode them into usable data, and then persist them in postgres. A transformer converts the raw chain data into a human friendly representation suitable for consumption in the API.
```

Comment:
Wondering if we might want to keep something stating that configuring event transformers in execute causes us to query the ethereum node for those logs and write them to the `public.event_logs` table (in addition to scanning that table and transforming them)

Reply:
I did not know it did that - yes it deserves a mention.

Reply:
I'm not sure what happened with the top here, I basically restored the top paragraph as I like it better. Blame turkey.